src/metabase/models/revision.clj

```diff
@@ -51,7 +51,7 @@
 (defmethod diff-str :default
   [model o1 o2]
   (when-let [[before after] (data/diff o1 o2)]
-    (diff-string (:name model) before after)))
+    (diff-string (name model) before after)))
```

Comment:
Toucan 1 models have a `:name` key that is basically the same as calling `name` on them. In Toucan 2 since models are keywords `name` gives you the same answer while `:name` does not work.

```clj
;;; Toucan 1
Card
#metabase.models.card.CardInstance{:table :report_card, :name "Card", ...}

(:name Card)
;; => "Card"

(name Card)
;; => "Card")

;;; Toucan 2
Card
:metabase.models.card/Card

(:name Card)
;; => nil

(name Card)
;; => "Card")
```